src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+function averageAge(ages) {
+  const agesSum = ages.reduce((sum, age) => sum + age, 0);
+
+  return agesSum / ages.length || 0;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person =>
+    person.sex === 'm'
+    && (century
+      ? Math.ceil(person.died / 100) === century
+      : true
+    ));
+
+  const mAges = men.map(man => man.died - man.born);
```

Comment:
is that what is stored inside the variable?

![image](https://github.com/mate-academy/js_average-ages/assets/103033758/9710ea85-5b7e-45de-8a6c-a9c2bc0fbd45)